# Translations template for designate.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the designate project.
#
# Translators:
# Andi Chandler <andi@gowling.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Designate\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-03-16 06:05+0000\n"
"PO-Revision-Date: 2015-03-13 18:48+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: English (United Kingdom) (http://www.transifex.com/projects/p/"
"designate/language/en_GB/)\n"
"Language: en_GB\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: designate/dnsutils.py:47
#, python-format
msgid "Failed to deserialize packet from %(host)s:%(port)d"
msgstr ""

#: designate/dnsutils.py:204
msgid "There was a problem with the AXFR"
msgstr ""

#: designate/service.py:339
#, python-format
msgid "Unhandled exception while processing request from %(host)s:%(port)d"
msgstr "Unhandled exception while processing request from %(host)s:%(port)d"

#: designate/api/middleware.py:267
msgid "Missing context in request, please check."
msgstr "Missing context in request, please check."

#: designate/backend/impl_ipa/__init__.py:400
msgid ""
"Error: could not authenticate to IPA - please check for correct keytab file"
msgstr ""
"Error: could not authenticate to IPA - please check for correct keytab file"

#: designate/backend/impl_ipa/auth.py:55 designate/backend/impl_ipa/auth.py:60
#, python-format
msgid "caught kerberos exception %r"
msgstr "caught kerberos exception %r"

#: designate/manage/tlds.py:138
#, python-format
msgid "Number of errors: %d"
msgstr "Number of errors: %d"

#: designate/manage/tlds.py:141
#, python-format
msgid ""
"Error Lines:\n"
"%s"
msgstr ""
"Error Lines:\n"
"%s"

#: designate/mdns/handler.py:126
#, python-format
msgid "got exception while handling axfr request. Question is %(qr)s"
msgstr ""

#: designate/network_api/neutron.py:126
#, python-format
msgid "Failed calling Neutron %(region)s - %(endpoint)s"
msgstr "Failed calling Neutron %(region)s - %(endpoint)s"

#: designate/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "in fixed duration looping call"

#: designate/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "in dynamic looping call"

#: designate/openstack/common/service.py:268
msgid "Unhandled exception"
msgstr "Unhandled exception"

#: designate/pool_manager/service.py:311
msgid ""
"An unhandled exception in periodic recovery occurred.  This should never "
"happen!"
msgstr ""

#: designate/pool_manager/service.py:341
msgid ""
"An unhandled exception in periodic sync occurred.  This should never happen!"
msgstr ""
